src/particle/data/__init__.py

```diff
@@ -5,6 +5,6 @@
 # or https://github.com/scikit-hep/particle for details.
 
 try:
-    from importlib.resources import open_text
+    from importlib.resources import open_text  # type: ignore
 except ImportError:
-    from importlib_resources import open_text  # type: ignore
+    from importlib_resources import open_text
```

Comment:
This is messy. The standard library got `importlib.resources ` in 3.7 and `open_text` in 3.8 or something like that, so MyPy is not always happy. This appeases the pre-commit run of mypy, so that's enough for now.